docs/development.md

```diff
@@ -1,32 +1,136 @@
 # Local Development
 
+This document will explain how to set up Wikijump on your machine for local development.
+
+## Environment
+
+### Operating system
+
+To run Wikijump, you will need to be running Linux.
+
+If you are running Windows, you have two options:
+
+* Install a Virtual Machine (VM) running Linux, for example using [VirtualBox](https://www.virtualbox.org/), and work from inside it as if it were Linux
+* (Recommended) Install [Windows Subsystem for Linux 2 (WSL2)](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
+
+In either case, when choosing a Linux distribution, Ubuntu is fine.
+
+### Connecting to localhost
+
+You will need edit your system's HOSTS file so that when you navigate to https://www.wikijump.test, your browser knows that you want to connect to your own virtual server and not somewhere on the internet. On Linux, your HOSTS file is `/etc/hosts`. On Windows, it is `C:\Windows\System32\drivers\etc\hosts`, but in order to edit it you will need to be running an editor (e.g. Notepad) as administrator. If you are running WSL2, edit your Windows HOSTS file; ignore your Linux HOSTS file.
+
+Add the following lines to your HOSTS file:
+
+```
+127.0.0.1 localhost
+127.0.0.1 wikijump.test
+127.0.0.1 www.wikijump.test
+127.0.0.1 profiles.wikijump.test
+127.0.0.1 template-en.wikijump.test
+127.0.0.1 wjfiles.test
+127.0.0.1 www.wjfiles.test
+```
```

Comment:
Would also add equivalents for `::1` ([IPv6](https://scuttle.atlassian.net/browse/WJ-172))

Also noting the order / formatting I use:
```
# Development
127.0.0.1                   wikijump.test               Wikijump
127.0.0.1                   www.wikijump.test           Wikijump
127.0.0.1                   wjfiles.test                Wikijump
127.0.0.1                   profiles.wikijump.test      Wikijump
127.0.0.1                   template-en.wikijump.test   Wikijump
127.0.0.1                   sandbox.wikijump.test       Wikijump
127.0.0.1                   scp-wiki.wikijump.test      Wikijump
::1                         wikijump.test               Wikijump
::1                         www.wikijump.test           Wikijump
::1                         wjfiles.test                Wikijump
::1                         profiles.wikijump.test      Wikijump
::1                         template-en.wikijump.test   Wikijump
::1                         sandbox.wikijump.test       Wikijump
::1                         scp-wiki.wikijump.test      Wikijump
```